src/aap_eda/api/views/activation.py

```diff
@@ -224,7 +225,7 @@ def enable(self, request, pk):
                 "modified_at",
             ]
         )
-        start_activation(activation_id=pk)
+        start_activation(KLASS_NAME, pk)
```

Comment:
@bzwei do you want to use kwargs style as opposed to positional args for your functions. I am guessing earlier it was a kwargs style function.

Reply:
should not matter, unless we have a policy to always use kwargs